src/map/status.c

```diff
@@ -1560,6 +1560,42 @@ static int status_fixed_revive(struct block_list *bl, unsigned int per_hp, unsig
 	return 1;
 }
 
+/**
+ * a sub-function from 'status_check_skilluse' function, which can be use for plugin manipulation
+ * check 'disabled_skills' flag settings from `db\(pre-)re\map_zone_db.conf`
+ * @param src        The source object can be BL_PC, BL_MOB, BL_PET, BL_HOM, BL_MER, BL_ELEM type
+ * @param st         The status_data of the source object, use to check MD_BOSS of BL_MOB type
+ * @param skill_id   The skill ID use
+ * @return false if the skill ID for the src is denied, true otherwise
+ **/
```

Comment:
Please separate the parameter list and the end of the comment block with an empty comment line.

Reply:
https://github.com/HerculesWS/Hercules/commit/bfa423fea55aa5659506cf85e6c418346804ac89#diff-759fea9dcd40522e86217dd352aadd687488dff49c67aa880610d4aeaaeb89c0R10941

nope, look at Haru's commit
well yes the param should leave an empty line, but there is no empty line between comment block and function

Reply:
Well, since our style guide doesn't specify this in detail I won't argue about that. :smile: